imports/zones/client.lua

```diff
@@ -261,6 +258,21 @@ local function convertToVector(coords)
     return coords
 end
 
+local function setDebug(self, bool, colour)
+    if not bool and insideZones[self.id] then
+        insideZones[self.id] = nil
+    end
+
+    self.debugColour = bool and vec4(colour?.r or self.debugColour?.r or 255, colour?.g or self.debugColour?.g or 42, colour?.b or self.debugColour?.b or 24, colour?.a or self.debugColour?.a or 100) or nil
+    Zones[self.id].debugColour = self.debugColour
```

Comment:
As `Zones[self.id] == self` there shouldn't be a need to reassign this property.

Reply:
Yeah I wasn't sure if it would update it